[2024-11-26T02:15:01.504+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-26T02:15:01.522+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_quarterly_performance_reports.ingest_performance_reports_task manual__2024-11-26T02:14:55.289730+00:00 [queued]>
[2024-11-26T02:15:01.531+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_quarterly_performance_reports.ingest_performance_reports_task manual__2024-11-26T02:14:55.289730+00:00 [queued]>
[2024-11-26T02:15:01.532+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-26T02:15:01.547+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): ingest_performance_reports_task> on 2024-11-26 02:14:55.289730+00:00
[2024-11-26T02:15:01.556+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=7942) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-26T02:15:01.558+0000] {standard_task_runner.py:72} INFO - Started process 7944 to run task
[2024-11-26T02:15:01.556+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ETL_quarterly_performance_reports', 'ingest_performance_reports_task', 'manual__2024-11-26T02:14:55.289730+00:00', '--job-id', '321', '--raw', '--subdir', 'DAGS_FOLDER/project_dag.py', '--cfg-path', '/tmp/tmpqvjdo3no']
[2024-11-26T02:15:01.559+0000] {standard_task_runner.py:105} INFO - Job 321: Subtask ingest_performance_reports_task
[2024-11-26T02:15:01.610+0000] {task_command.py:467} INFO - Running <TaskInstance: ETL_quarterly_performance_reports.ingest_performance_reports_task manual__2024-11-26T02:14:55.289730+00:00 [running]> on host 6a246d8d590f
[2024-11-26T02:15:01.921+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_quarterly_performance_reports' AIRFLOW_CTX_TASK_ID='ingest_performance_reports_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-26T02:14:55.289730+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-26T02:14:55.289730+00:00'
[2024-11-26T02:15:01.923+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-26T02:15:02.615+0000] {logging_mixin.py:190} INFO - Error: column "time_period_amount" of relation "amountdescriptions" does not exist
LINE 1: ... AmountDescriptions (amount_desc_id, description, time_perio...
                                                             ^
[2024-11-26T02:15:25.371+0000] {logging_mixin.py:190} INFO - Error: column "benefit_type" of relation "activities" does not exist
LINE 1: ...end_date, national_objective, responsible_entity, benefit_ty...
                                                             ^
[2024-11-26T02:15:26.866+0000] {logging_mixin.py:190} INFO - Error: column "activity_number" of relation "performancereports" does not exist
LINE 1: INSERT INTO PerformanceReports (activity_id, Activity_Number...
                                                     ^
[2024-11-26T02:15:26.868+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-26T02:15:26.878+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-26T02:15:26.879+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ETL_quarterly_performance_reports, task_id=ingest_performance_reports_task, run_id=manual__2024-11-26T02:14:55.289730+00:00, execution_date=20241126T021455, start_date=20241126T021501, end_date=20241126T021526
[2024-11-26T02:15:26.937+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-26T02:15:26.961+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-26T02:15:26.966+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
